components/gitpod-cli/cmd/top.go

```diff
@@ -35,45 +34,49 @@ type topData struct {
 var topCmd = &cobra.Command{
 	Use:   "top",
 	Short: "Display usage of workspace resources (CPU and memory)",
-	Run: func(cmd *cobra.Command, args []string) {
+	RunE: func(cmd *cobra.Command, args []string) (err error) {
```

Comment:
```suggestion
	RunE: func(cmd *cobra.Command, args []string) error {
```

In this case, I prefer to not use `named return` for all files, it will make result more clear